### DISCUSSION TITLE
Updates that are not version updates

### DISCUSSION THREAD

**Discussion by sudwhiwdh** [borgbase/vorta#1702] [FAQ] [answered]
Hi, from time to time GNOME software shows me updates for Vorta, but they are not version updates. Where exactly do these updates come from and what do they update? Unfortunately there is no changelog for this ...

**Comment by real-yfprojects** [comment]
Did you install from flathub?

**Reply by sudwhiwdh** [reply]
Yes exactly, it's the version from Flathub.

**Reply by real-yfprojects** [reply, marked as answer]
A new build might be released on flathub when the bundled dependencies are updated.

**Reply by sudwhiwdh** [reply]
GNOME Software does not display any information about this. Only that it is an update for Vorta.

**Reply by real-yfprojects** [reply]
Builds can be tracked here: https://buildbot.flathub.org/#/apps/com.borgbase.Vorta
